src/Apps/Order/Utils/scrollToFormikErrors.tsx

```diff
@@ -0,0 +1,33 @@
+import { useFormikContext } from "formik"
+import { useEffect } from "react"
+
+const getFirstFieldErrorName = errors => {
+  if (Object.keys(errors).length === 0) return
+
+  const prefix = Object.keys(errors)[0]
+
+  const field = prefix && Object.keys(errors[prefix])[0]
+
+  return `${prefix}.${field}`
+}
+
+export const ScrollToFieldError = () => {
```

Comment:
I imagine this is a typo but still wanna flag :) Can we update this so that it follows more standard function naming conventions?  (`CapsCase` is typically reserved for classes, components, interfaces, types and enums, not regular functions)

Reply:
Also, might want to name file name after the main named export -- so `scrollToFieldError.ts` -- and then if more formik utils are added, we can rename the file to something more general. That way, things are 1:1.  

Otherwise, looks good! If this is used more frequently we might want to consider elevating it up to the main Utils folder (or something.)

Reply:
However you think is best! (Or not at all is totally fine -- just wanted to call it out, its super minor)